web/api/v1/me/licenses-certifications.cshtml

```diff
@@ -115,6 +115,7 @@
 
                 // We don't want the ugly default asp.net message for file-size-exceeded
                 // As of #454 (https://github.com/joshdanielson/Loconomics/issues/454#issuecomment-37782181)
+                //TODO: fix for localisation of strings
```

Comment:
The problem with this error message is that is provided by 'asp.net', so it depends in how is localized by the .net framework on every language. At some points, we force this text just to emulate it (line 93 this file) so we trigger this same code that replaces the built-in message by something clear for the user.
The ideal is being able to detect an error code or exception type, but since we did it this way I 'suppose' is because we found there is no one.

Anyway, since the back-end is to be replaced and this is .net specific, this should not be a problem soon, so we should take care to detect system error codes / errno rather if possible when replacing system errors by something customized.

Reply:
I'll remove this one. seems complicated and pretty edge case.